cypress/objects/common.js

```diff
@@ -0,0 +1,40 @@
+class Common {
+    
+    getSpecificUrl(url) {
+        cy.visit(url); 
+    }
```

Comment:
there is no point of having this function, as it just takes url and pass it to another function, you can call cy.visit(url) with same results